### PR TITLE
ExperiSci tweaks

### PR DESCRIPTION
**Pull request by PolishUser**
## About The Pull Request

This is the first step in tweaking ExperiSci.
This PR is going to add a radio to the console which is going to message respective department's radio channels connected with the node being researched. Furthermore, this PR is going to revert 50K general research point bomb and previous research point prices before Experisci. In addition, experiment discounts and requirements are gone.

## Changelog
:cl:
add: Revert 50k point scibomb
add: Nodes upon research announce research in respective department's radio channels related to the node
del: Remove experisci requirements and discounts
balance: Reduce point requirement for some nodes
:cl:

### CODE REVIEW

**Comment by Burgermancoding** [comment]
good stuff

**Comment by TheChosenEvilOne** [comment]
stuff is good

**Comment by AcapitalA** [comment]
It has come to my attention that all these variated antiquities you are performing are seemingly fitting this repository quite well.

**Comment by Sparkezel** [comment]
Best video I've ever seen!

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -743,6 +763,7 @@
 		"skill_station",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:766]:
```suggestion
```
Spammy and not interesting enough

File: code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -778,6 +800,7 @@
 		"borg_upgrade_rped",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2000)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_ENGINEERING,RADIO_CHANNEL_AI_PRIVATE)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:803]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_AI_PRIVATE)
```
No engineering, the borg can already hear everything and engies don't care

File: code/modules/research/techweb/all_nodes.dm [hunk 3]
```diff
@@ -793,6 +816,7 @@
 		"borg_upgrade_surgicalprocessor",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2000)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_MEDICAL,RADIO_CHANNEL_AI_PRIVATE)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:819]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_AI_PRIVATE)
```
Ditto

File: code/modules/research/techweb/all_nodes.dm [hunk 4]
```diff
@@ -842,6 +867,7 @@
 		"vendatray",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_ENGINEERING)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:870]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_ENGINEERING)
```

File: code/modules/research/techweb/all_nodes.dm [hunk 5]
```diff
@@ -908,6 +937,7 @@
 		"seccamera",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2000)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_ENGINEERING)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:940]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_SCIENCE, RADIO_CHANNEL_SERVICE, RADIO_CHANNEL_ENGINEERING)
```
Library + HoP

File: code/modules/research/techweb/all_nodes.dm [hunk 6]
```diff
@@ -955,8 +986,8 @@
 		"arcade_orion",
 		"slotmachine",
 	)
-	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 3250)
-	discount_experiments = list(/datum/experiment/physical/arcade_winner = 3000)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_SERVICE)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:990]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_SERVICE)
```

File: code/modules/research/techweb/all_nodes.dm [hunk 7]
```diff
@@ -972,13 +1003,15 @@
 		"vendor",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 1000)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_ENGINEERING)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:1006]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_SCIENCE, RADIO_CHANNEL_ENGINEERING, RADIO_CHANNEL_SECURITY)
```

File: code/modules/research/techweb/all_nodes.dm [hunk 8]
```diff
@@ -972,13 +1003,15 @@
...
 
 /datum/techweb_node/telecomms
 	id = "telecomms"
 	display_name = "Telecommunications Technology"
 	description = "Subspace transmission technology for near-instant communications devices."
 	prereq_ids = list("comptech", "bluespace_basic")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_ENGINEERING)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:1014]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_ENGINEERING)
```

File: code/modules/research/techweb/all_nodes.dm [hunk 9]
```diff
@@ -1039,6 +1074,7 @@
 		"scan_console",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:1077]:
```suggestion
```
Spam + already have disks roundstart

File: code/modules/research/techweb/all_nodes.dm [hunk 10]
```diff
@@ -1361,7 +1412,7 @@
 		"pyro_Grenade",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
-	required_experiments = list(/datum/experiment/explosion/maxcap)
+	channel_tag = list(RADIO_CHANNEL_SECURITY)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:1415]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_MEDICAL, RADIO_CHANNEL_SECURITY)
```
It's actually just grenades

File: code/modules/research/techweb/all_nodes.dm [hunk 11]
```diff
@@ -1397,6 +1450,7 @@
 		"mech_gravcatapult",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:1453]:
```suggestion
```
Single stuff spam

File: code/modules/research/techweb/all_nodes.dm [hunk 12]
```diff
@@ -1546,6 +1607,7 @@
 		"mech_syringe_gun",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_MEDICAL)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:1610]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_SCIENCE)
```

File: code/modules/research/techweb/all_nodes.dm [hunk 13]
```diff
@@ -1534,6 +1594,7 @@
 		"mech_thrusters",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_ENGINEERING)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:1597]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_SCIENCE)
```

File: code/modules/research/techweb/all_nodes.dm [hunk 14]
```diff
@@ -1955,7 +2044,7 @@
 	)
 
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
-	required_experiments = list(/datum/experiment/scanning/points/slime/hard)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:2047]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_SCIENCE, RADIO_CHANNEL_MEDICAL)
```
Alien surgery tools

File: code/modules/research/techweb/all_nodes.dm [hunk 15]
```diff
@@ -2006,6 +2096,7 @@
 		"suppressor",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 10000)
+	channel_tag = list(RADIO_CHANNEL_SCIENCE,RADIO_CHANNEL_SECURITY)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:2099]:
```suggestion
	channel_tag = list(RADIO_CHANNEL_SCIENCE, RADIO_CHANNEL_SECURITY, RADIO_CHANNEL_AI_PRIVATE, RADIO_CHANNEL_MEDICAL)
```
AI cam upgrades + rapid syringe gun

File: code/modules/research/techweb/all_nodes.dm [hunk 16]
```diff
@@ -2041,6 +2133,7 @@
 		"rolling_table",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	channel_tag = list(RADIO_CHANNEL_COMMON)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:2136]:
```suggestion
```

Comment (Canohaaaaaaa) [reply code/modules/research/techweb/all_nodes.dm:2136]: Or maybe service perhaps?

File: code/modules/research/techweb/all_nodes.dm [hunk 17]
```diff
@@ -2092,6 +2188,7 @@
 		"super_sticky_tape",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+	channel_tag = list(RADIO_CHANNEL_COMMON)
```
Comment (Canohaaaaaaa) [inline code/modules/research/techweb/all_nodes.dm:2191]:
```suggestion
```

**Comment by Canohaaaaaaa** [comment]
Also explain the point changes in the PR body and add your radio changes to the changelog (even if it doesn't work for now)

File: code/modules/research/techweb/all_nodes.dm (hunk 4 above)
Comment (Canohaaaaaaa) [reply code/modules/research/techweb/all_nodes.dm:870]: Also service

**Pull request merged**